gnmi_cli_py/py_gnmicli.py

```diff
@@ -430,8 +441,15 @@ def subscribe_start(stub, options, req_iterator):
               print('gNMI Error '+str(response.error.code)+\
                 ' received\n'+str(response.error.message) + str(response.error))
           elif response.HasField('update'):
-              print(response)
-              update_count = update_count+1
+              if filter_event != "":
```

Comment:
>filter_event [](http://example.com/codeflow?start=17&length=12)

Normally we treat `None` and empty string the same way.

Reply:
If `filter_event_regex` is empty string, it will match everything. Seem this is weird usage. Do you want to stop user to do it?

Reply:
Added check for empty string